src/store/actions/set_user.js

```diff
@@ -10,13 +10,22 @@ export default firestoreAction(async ({ bindFirestoreRef, unbindFirestoreRef },
     throw new Error('Invalid user');
   };
 
-  // Check if user is valid Google account
+  // Check if user is valid Google account and within the knowit.no-domain
   if (!user || !user.email) rejectAccess();
-
+ 
   // Check if user is whitelisted
   const userRef = db.collection('users').doc(user.email);
-  const documentSnapshot = await userRef.get();
-  if (!documentSnapshot.exists) rejectAccess();
+  let documentSnapshot = await userRef.get();
+
+  if (!documentSnapshot.exists) {
+    // Add user if it does not exist
```

Comment:
Er jeg blind eller mangler du en sjekk på om det er knowit.no domenet?

Reply:
Det er fjernet med vilje og dokumentert i README. For microsoft setter man opp TENENT og styrer det sånn, Google støtter ikke dette - men vil ikke hardkode noe inn. I Knowit vil vi uansett bare bruke Microsoft fremover.

Reply:
Et annet argument er at vi ønsker mulighet for eksterne brukere/partnere som logger på med eposter fra andre domener.

Reply:
haha, må begynne å lese Readme.... Det høres jo kult ut! Takk for forklaringen :D